chain/substreams/src/trigger.rs

```diff
@@ -217,6 +217,11 @@ where
                         logger,
                     );
 
+                    data.insert(
+                        Word::from("id"),
+                        decode_value(&codec::value::Typed::String(entity_id)).unwrap(),
```

Comment:
you could just use `Value::from(&entity_id)` here and save a trip through `decode_value`

Reply:
Good point, I made the changes.